### DISCUSSION TITLE
Restore AKS Disks/persistent volumes in another availability zone

### DISCUSSION THREAD

**Discussion by pranitha9** [vmware-tanzu/velero#3731] [Velero featured]
I am working for Backup and restore of Azure kubernetes services Disks (persistent volumes) for DR scenario.

The thing is I am able to Backup AKS PVs and restore them in same availability zone by using Velero.

but considering the DR scenario - if one availability zone crashes then we need to restore AKS PVs in another AZ and able to attach them to AKS pods.

please suggest the way how to achieve this DR scenario using Velero.

Thank you. Happy to provide more details if needed.

**Comment by Fabien1104** [comment]
Hello, This is a current limitation of Velero. 
From the current documentation: "_Volume snapshots are limited by where your provider allows you to create snapshots. For example, AWS and Azure do not allow you to create a volume snapshot in a different region than where the volume is located. If you try to take a Velero backup using a volume snapshot location with a different region than where your cluster’s volume is, the backup will fail_._ "
As of today, I didn't see any coming features which would permit to answer to that DR scenario with disks. 
Fabien
